setup.cfg

```diff
@@ -0,0 +1,6 @@
+[aliases]
+test              = pytest
```

Comment:
Was the white spacing here done on purpose?

Reply:
Fixed